extra/plot_lib.py

```diff
@@ -1 +0,0 @@
-../plot_lib.py
```

Comment:
Wait, what?

Reply:
This was a broken symlink... I.e. `./plot_lib.py` didn't exist for me...